ci-operator/config/openshift-assisted/assisted-installer-ui/openshift-assisted-assisted-installer-ui-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  node18:
+    name: nodejs-18-minimal
+    namespace: edge-infrastructure
+    tag: latest
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  openshift_ubi-micro_9:
+    name: ubi-micro
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_path: apps/assisted-disconnected-ui/Containerfile
+  to: assisted-disconnected-ui
+promotion:
+  to:
+  - name: "4.19"
+    namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.19"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.19"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+zz_generated_metadata:
```

Comment:
I'd recommend to add at least one test in the `tests` section, so to have a starting job (maybe a simple linting/unit testing one) to verify that the CI setup is ok